src/main/kotlin/no/nav/syfo/cronjob/CronjobMetric.kt

```diff
@@ -3,11 +3,8 @@ package no.nav.syfo.cronjob
 import io.prometheus.client.Counter
 import no.nav.syfo.metric.METRICS_NS
 
-const val CRONJOB_BASE = "cronjob"
-const val CRONJOB_JOURNALFORING_BASE = "${CRONJOB_BASE}_journalforing"
-
-const val CRONJOB_JOURNALFORING_VARSEL_UPDATE = "${CRONJOB_JOURNALFORING_BASE}_update_count"
-const val CRONJOB_JOURNALFORING_VARSEL_FAIL = "${CRONJOB_JOURNALFORING_BASE}_fail_count"
+const val CRONJOB_JOURNALFORING_VARSEL_UPDATE = "cronjob_journalforing_update_count"
+const val CRONJOB_JOURNALFORING_VARSEL_FAIL = "cronjob_journalforing_fail_count"
```

Comment:
Endret til metrikk-navn uten gjenbruk av prefix for å gjøre det lettere å finne og unngå å slette disse metrikkene.